backend/src/main/java/org/eclipse/tractusx/puris/backend/common/edc/domain/model/SubmodelType.java

```diff
@@ -23,6 +23,9 @@
 public enum SubmodelType {
     DTR("none", "none"),
     ITEM_STOCK("urn:samm:io.catenax.item_stock:2.0.0#ItemStock", "$value"),
+    PRODUCTION("urn:samm:io.catenax.planned_production_output:2.0.0 ", "$value"),
```

Comment:
Please add Entity name after urns with "#EntityName"
Sidenote: the models can be found in the following repo: https://github.com/eclipse-tractusx/sldt-semantic-models

Reply:
sidenote: Please remove the spaces at the end of the type.